tests/fsharp/tests.fs

```diff
@@ -973,6 +973,46 @@ module CoreTests =
         | _ -> Assert.Fail (sprintf "'%s' and '%s' differ; %A" stderrPath stderrBaseline diffs2)
 #endif
 
+    [<Test>]
+    let ``typeinferenceWarning`` () = 
+        let cfg = testConfig "core/typeinferenceWarning"
```

Comment:
Normally we'd add a negNN.fs test for such a case? Is there any reason not to be doing that?

Reply:
Reason is that I have no idea what I’m doing but I still wanted to add a test ;)

Is there docs around that or can someone help?

Reply:
What is "negNN"?

Reply:
typecheck/sigs/neg101.fs etc. for example